src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -100,6 +102,15 @@ public override T GetInput<T>()
             return MessagePayloadDataConverter.Default.Deserialize<T>(this.serializedInput);
         }
 
+        /// <inheritdoc />
+        public override Guid NewGuid()
+        {
+            this.newGuidCounter++;
+            StringBuilder newGuidNameStringBuilder = new StringBuilder(this.InstanceId);
+            newGuidNameStringBuilder.Append(this.newGuidCounter);

```

Comment:
Using a `StringBuilder` to concatenate two strings is actually more overhead than doing string concatenation directly. I suggest simply:
```csharp
string guidNameValue = this.InstanceId + this.newGuidCounter.ToString();
this.newGuidCounter++;
```

Reply:
Hello @cgillum,

Great!

I will modify it!

Thank you!